sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -284,11 +288,11 @@ public JournalPersonDto getPersonForJournal(String uuid) {
 		if (detailedPerson != null) {
 			JournalPersonDto exportPerson = new JournalPersonDto();
 			exportPerson.setUuid(detailedPerson.getUuid());
-			exportPerson.setEmailAddress(detailedPerson.getEmailAddress());
+			exportPerson.setEmailAddress(detailedPerson.getEmailAddress(false));
```

Comment:
This one can no throw a SeveralNonPrimaryContactDetailsException, right? In that case, null should be returned here.

Reply:
It should not be the case, as all persons registered in journals should have the correct contact information.
Registering with multiple email addresses is not allowed.

Reply:
Yes, but that's a trap: For patient diaries it's different. And for patient diaries it can happen.